<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="g2v1">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>51</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icon/g2v1.jpg</normaloff>icon/g2v1.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>51</width>
      <height>101</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="g41">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>230</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icon/g4.jpg</normaloff>icon/g4.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>101</width>
      <height>101</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="g2v3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>230</y>
      <width>51</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icon/g2v2.jpg</normaloff>icon/g2v2.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>51</width>
      <height>101</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="g2v4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>330</y>
      <width>51</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icon/g2v4.jpg</normaloff>icon/g2v4.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>51</width>
      <height>101</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="g2v2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>330</y>
      <width>51</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icon/g2v3.jpg</normaloff>icon/g2v3.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>51</width>
      <height>101</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="g2h1">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>330</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icon/g2h1.jpg</normaloff>icon/g2h1.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>101</width>
      <height>51</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="g11">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>380</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icon/g1.jpg</normaloff>icon/g1.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>51</width>
      <height>51</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="g12">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>380</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icon/g1.jpg</normaloff>icon/g1.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>51</width>
      <height>51</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="g14">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>430</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icon/g1.jpg</normaloff>icon/g1.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>51</width>
      <height>51</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="g13">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>430</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icon/g1.jpg</normaloff>icon/g1.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>51</width>
      <height>51</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>210</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>solve</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>766</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
